pkgs/applications/networking/instant-messengers/signalbackup-tools/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib, stdenv, fetchFromGitHub, openssl, sqlite }:
+
+stdenv.mkDerivation rec {
+  pname = "signalbackup-tools";
+  version = "20220107";
+
+  src = fetchFromGitHub {
+    owner = "bepaald";
+    repo = pname;
+    rev = version;
+    sha256 = "sB8/xQgSORtwupcwSejKUhHoz04exdYS0ymefw9wXDQ=";
```

Comment:
sri hashes should retain the sha256- prefix.

Reply:
Done.